velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -496,32 +505,113 @@ class HiveIcebergTest : public HiveConnectorTestBase {
     return deleteFilePaths;
   }
 
+  template <typename T>
   std::vector<RowVectorPtr> makeVectors(
-      std::vector<int64_t> vectorSizes,
-      int64_t& startingValue) {
+      std::vector<T> vectorSizes,
+      T& startingValue) {
     std::vector<RowVectorPtr> vectors;
     vectors.reserve(vectorSizes.size());
 
     vectors.reserve(vectorSizes.size());
     for (int j = 0; j < vectorSizes.size(); j++) {
       auto data = makeContinuousIncreasingValues(
           startingValue, startingValue + vectorSizes[j]);
-      VectorPtr c0 = makeFlatVector<int64_t>(data);
+      VectorPtr c0 = makeFlatVector<T>(data);
```

Comment:
`makeContinuousIncreasingValues` takes only integer arguments. Add a check whether T is integral type here too.

Reply:
Could you please remove makeContinuousIncreasingValues() and replace it with makeSequenceValues<T>()?